packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -521,12 +534,18 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         const chunks = await tryFetchBlob<[string, string[]][]>(chunksBlobName) ?? [];
         const metadata = await tryFetchBlob<IContainerRuntimeMetadata>(metadataBlobName);
 
+        // enableGC in metadata is introduced with v1 in the metadata blob. Force to false before that.
+        const documentEnableGC = gcEnabled(metadata);
```

Comment:
This will override the value in runtimeOptions (if any) right? Can you please add that to the comment.